lib/media.js

```diff
@@ -0,0 +1,251 @@
+'use strict';
+
+const _ = require('lodash'),
+  path = require('path'),
+  files = require('nymag-fs'),
```

Comment:
i _think_ we try not to use this package anymore but i dont remember what we use instead